src/System.Drawing.Primitives/tests/SizeFTests.netcoreapp.cs

```diff
@@ -23,19 +39,22 @@ public partial class SizeFTests
         [InlineData(float.MinValue, float.MaxValue)]
         public void MultiplicationTest(float value1, float value2)
         {
-            SizeF sz1 = new SizeF(value1, value1);
-            SizeF sz2 = new SizeF(value2, value2);
-            SizeF mulExpected;
+            MultSizeFFloatTester(value1, 1.0f, value2);
```

Comment:
All the previous test cases for operator * or operator / treated width and height in symmetric way. Hence changed the test cases to test effect of operator on width and height effects separately.

While reverting the code for `operator /` for type `Size`, I mistakenly ended up writing `left._width / right` for both `width` and `height` params for creating new instance of `Size` returned from the `operator /`. That prompted me to make the above changes so that width and height are tested separately.